CoreGraphFile/src/au/gov/asd/tac/constellation/graph/file/open/OpenFilePlugin.java

```diff
@@ -24,110 +24,51 @@
 import au.gov.asd.tac.constellation.plugins.parameters.PluginParameters;
 import au.gov.asd.tac.constellation.plugins.templates.PluginTags;
 import au.gov.asd.tac.constellation.plugins.templates.SimpleReadPlugin;
+import au.gov.asd.tac.constellation.utilities.gui.filechooser.FileChooser;
 import java.io.File;
-import javax.swing.JFileChooser;
-import javax.swing.SwingUtilities;
-import org.openide.filesystems.FileUtil;
-import org.openide.loaders.DataObject;
+import org.openide.filesystems.FileChooserBuilder;
+import org.openide.util.HelpCtx;
 import org.openide.util.NbBundle.Messages;
-import org.openide.util.UserCancelException;
 import org.openide.util.lookup.ServiceProvider;
-import org.openide.windows.TopComponent;
-import org.openide.windows.WindowManager;
 
 /**
- * Open a file
+ * Open a file.
  *
  * @author canis_majoris
+ * @author sol695510
  */
 @ServiceProvider(service = Plugin.class)
 @Messages("OpenFilePlugin=Open File")
 @PluginInfo(pluginType = PluginType.IMPORT, tags = {PluginTags.LOW_LEVEL})
 public class OpenFilePlugin extends SimpleReadPlugin {
 
-    private boolean running;
-    private static File currentDirectory = null;
+    private static File savedDirectory = FileChooser.DEFAULT_DIRECTORY;
 
-    private static void setCurrentDirectory(final File currentDir) {
-        currentDirectory = currentDir;
+    private static final String TITLE = "Open";
+
+    @Override
+    public HelpCtx getHelpCtx() {
+        return new HelpCtx(this.getClass().getName());
```

Comment:
I'm a little confused by this inclusion here. Not sure we were creating a help page links here(?)

Reply:
This was in here ages ago before the new help. I assume its probably not needed now from your comment so I can remove it.